nulib/meadow
# Terraform fixity step function invocation

# Summary 

Trigger fixity step function with S3 Bucket notification + lambda due to bug in triggering with CloudTrail + EventBridge for large files

# Specific Changes in this PR

- Terraform s3 bucket notification
- Terraform `execute-fixity-function` lambda
- Remove CloudTrail and event bridge infrastructure from Terraform

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test


- Preform ingest in UI
- Perform file set upload in UI
- Make sure pipeline completes successfully (checksums and all other actions)

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
